test/integration/load_stats_integration_test.cc

```diff
@@ -144,18 +148,57 @@ class LoadStatsIntegrationTest : public HttpIntegrationTest,
     loadstats_stream_ = fake_loadstats_connection_->waitForNewStream(*dispatcher_);
   }
 
+  void mergeLoadStats(envoy::api::v2::LoadStatsRequest& loadstats_request,
+                      const envoy::api::v2::LoadStatsRequest& local_loadstats_request) {
+    ASSERT(loadstats_request.cluster_stats_size() <= 1);
+    ASSERT(local_loadstats_request.cluster_stats_size() <= 1);
+
+    if (local_loadstats_request.cluster_stats_size() == 0) {
+      return;
+    } else if (loadstats_request.cluster_stats_size() == 0) {
+      loadstats_request.CopyFrom(local_loadstats_request);
+      return;
+    }
+
+    auto local_cluster_stats = local_loadstats_request.cluster_stats(0);
+    auto* cluster_stats = loadstats_request.mutable_cluster_stats(0);
+
+    cluster_stats->set_total_dropped_requests(cluster_stats->total_dropped_requests() +
```

Comment:
Shouldn't this be a `+=` operation?

Reply:
Isn't this the equivalent of += with proto?  I thought set_ was the only way to set it...